types/underscore/underscore-tests.ts

```diff
@@ -18,12 +18,26 @@ interface AugmentedList extends _.List<StringRecord> {
     notAListProperty: boolean;
 }
 
+// tslint:disable-next-line:interface-over-type-literal
+type AugmentedListLiteral = {
```

Comment:
Unfortunately I had trouble isolating what makes the JQuery interface special as compared to other interfaces that didn't run into this. I attempted to replicate this behavior in tests by using interfaces (going so far as to try generics and an interface with the same name as a namespace) but was unable to do so. However, the use of type literals still allowed for a comparable repro to help prevent regressions in the future. Also, unlike with interfaces this is a consistently reproduceable issue for types declared this way in more versions of TS than just 4.0, so I think it's still useful to test this case specifically since it's a pattern that's discouraged but still allowed.